docs/source/getting_started_run_fabric.rst

```diff
@@ -0,0 +1,20 @@
+Getting Started - Run Fabric
+=========================
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   Running a Test Network <test_network>
+   write_first_app
+
+
+Now that you have installed Fabric and Samples, you can start running Fabric.  (If you have not installed all the required software, follow the instructions on :doc:`getting_started` first).
```

Comment:
I feel like a broken record, but there is no reason to capitalize "samples" here. It is not a proper noun. Also, I can't see a reason this second sentence is inside a parenthesis. It's a perfectly reasonable sentence to have here.

Reply:
done.